src/ansys/aedt/core/visualization/report/common.py

```diff
@@ -578,7 +578,7 @@ def polyline(self):
             try:
                 self._props["context"]["polyline"] = self.traces[0].properties["Geometry"]
             except Exception:
-                pass
+                self._post._app.logger.debug("Something went wrong while processing polyline.")
```

Comment:
Following the EAFP principle, some parts of the code have this `try: except: pass` clause instead of an `if else` check, because is faster. If we put a log entry for every Exception hit, we might flood the logger. 
Although I haven't check in detail this part of the code, I rise this concern: are you sure that this is an Exception worth to be logged?

Reply:
I understand your point but the pattern `try: ... except: pass` is generally not recommended. Indeed, this pattern:
- swallows all exceptions, unless you add the type of exception you want to handle but that's not the case here;
- hides failures as there is no indication or feedback on what went wrong, which makes it harder to detect and fix a problem.

Here, I'm just logging at debug level so this shouldn't hinder too much unless you want to log at debug level. And, in that case, I feel like having information would help solving the problem even if the log are more verbose.

Finally, we have to use `bandit` and `safety` to upgrade our vulnerability handling. The pattern `try: ... except: pass` is reported by `bandit` as an issue of "low" severity but still an issue.

@Samuelopez-ansys @maxcapodi78 @gmalinve What are your thoughts on this ?

Reply:
@Alberto-DM @SMoraisAnsys 
I agree with Sebastien. 
If we know upfront that this part of the code is not intended to be repeated until it suceeds, where passing is good enough, then I agree on the logging regardless.

Reply:
That is correct. We should avoid the try/except/pass when possible especially as it can have serious side effects.
With try/except we have to catch and handle specific exceptions as much as possible.

@Alberto-DM's concerns about performance makes total sense and is a very valid point.
So here, it is a good practice to only use the debug mode for the logger to not overload the logger as mentioned.
It could be a pattern that we keep in mind for other similar case.